lessons/ro/chapter_10.yaml

```
@@ -0,0 +1,145 @@
+- title: Capitolul 10 - I/o
+  content_markdown: >
+    În informatică, `I/O` este o abreviere pentru operația de `Input/Output`.
+
+    `Input-ul` reprezintă datele primite de calculator și algoritm,
+    iar `output-ul` reprezintă rezultatul generat pe baza `input-ului`.
+
+    Gândește-te la `I/O` ca la un flux de informații.
+
+    Un sistem de calcul fără output este aproape inutil.
+    Va rula întotdeauna același cod pe aceleași date și, astfel, va produce același rezultat.
+
+- title: Rulează local
+  content_markdown: >
+    În acest capitol, Playground-ul va fi doar un suport de cod pentru tine :(.
+
+    Deoarece majoritatea programelor `I/O` sunt proiectate pentru a compila pe o mașină locală
+    (calculatorul tău :) ), ia în considerare configurarea unui mediu Rust pe computerul personal și
+    familiarizează-te cu terminalul.
+
+    De asemenea, consideră utilizarea unui `IDE`, cum ar fi `VS Code` sau `RustRover`
+    și familiarizați-vă cu [terminalul](https://www.youtube.com/watch?v=lZ7Kix9bjPI).
```

```suggestion
    și familiarizează-te cu [terminalul](https://www.youtube.com/watch?v=lZ7Kix9bjPI).
```
Verbele ar trebui sa fie la aceeasi persoana intr-o fraza.